services/apis/pidgin/pidginProps.js

```diff
@@ -2,11 +2,6 @@
  * pidgin Properties
```

Comment:
none of these props are actually used in the pidgin tests, they are examples that should be removed